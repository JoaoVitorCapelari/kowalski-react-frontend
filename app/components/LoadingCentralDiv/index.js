/**
*
* LoadingCentralDiv
*
*/

import React from 'react';
import { RingLoader } from 'react-spinners';
import CentralDiv from '../CentralDiv';

function LoadingCentralDiv() {
  return (
    <CentralDiv>
      <RingLoader />
    </CentralDiv>
  );
}

LoadingCentralDiv.propTypes = {

};

export default LoadingCentralDiv;
