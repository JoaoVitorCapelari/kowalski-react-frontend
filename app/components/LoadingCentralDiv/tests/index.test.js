import React from 'react';
import { shallow } from 'enzyme';

import LoadingCentralDiv from '../index';

describe('<LoadingCentralDiv />', () => {
  it('renders', () => {
    const renderedComponent = shallow(
      <LoadingCentralDiv />
    );
    expect(renderedComponent.html()).toMatchSnapshot();
  });
});
