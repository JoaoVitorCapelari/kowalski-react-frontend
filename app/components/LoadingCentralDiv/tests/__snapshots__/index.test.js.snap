// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadingCentralDiv /> renders 1`] = `"<div class=\\"CentralDiv__CentralDiv-jeadek bbknNc\\"><div class=\\"css-jxiqlq\\"><div class=\\"css-1lt1sep\\"></div><div class=\\"css-1vahj0q\\"></div></div></div>"`;
